docs/administration/segregation.md

```diff
@@ -22,63 +18,62 @@ In OpenCTI, data is segregated based on knowledge marking. The diagram provided
+## Manage markings
 
-For more details, the diagram provided below illustrates how are categorized the marking definitions:
+### Create new markings
 
-![TLP diagram](assets/tlp-diagram.png)
+To create a marking, you must first possess the capability `Manage marking definitions`. For further information on user administration, please refer to the [Users and Role Based Access Control page](./users.md).
 
-### Create new markings
+Once you have access to the settings, navigate to "Settings > Security > Marking Definitions" to create a new marking.
 
-In order to create a marking, you must first have the ability to access the Settings tab. For example, a user who is in a group with the role of Administrator can bypass all capabilities or a user who is in a group with the role that has `Access administration` checked can access the Settings tab. For more details about user administration, please see the [Users and Role Based Access Control page](./users.md).
+A marking consists of the following attributes:
 
-![Authorize access administration](assets/authorize-access.gif)
+- Type: Specifies the marking group to which it belongs.
+- Definition: The name assigned to the marking.
+- Color: The color associated with the marking.
+- Order: Determines the hierarchical order among markings of the same type.
 
-Once you have access to the settings, you can create a new marking in `Security` > `Marking Definitions`
+![Create a marking animation](assets/create-marking.gif)
 
-A marking has:
 
-- a type,
-- a definition,
-- a color, and
-- an order.
+### Allowed marking  
 
-![Create a marking animation](assets/create-marking.gif)
+The configuration of authorized markings for a user is determined at the [Group](users.md#group-section) level. To access entities and relationships associated with specific markings, the user must belong to a group that has been granted access to those markings.
 
-### Allowed marking definitions 
+There are two ways in which markings can be accessed:
 
-In order for all users in a group to be able to see entities and relationships that have specific markings on them, allowed markings can be checked when updating a group:
+- The user is a member of a group that has been granted access to the marking.
+- The user is a member of a group that has access to a marking of the same type, with an equal or higher hierarchical order.
 
-![Update allowed markings in a group](assets/update-group-marking.png)
+!!! note "Access to an object with several markings"
 
-### Default marking definitions
+    Access to all markings attached to an object is required in order to access it (not only one).
 
-To apply a default marking when creating a new entity or relationship, you can choose which marking to add by default from the list of allowed markings. You can add *only one marking per type*, but you can have multiple types.
+!!! tip "Automatically grant access to the new marking"
```

Comment:
I didn't understand why it has a copy/paste option. 

I understand the topic and explanation.

Reply:
![image](https://github.com/OpenCTI-Platform/docs/assets/168524367/24886ba2-57ab-4dc7-94bb-80ea8f2caf62)

Reply:
In terms of format/display ?